Automattic/wp-calypso
# Added "Defensive mode enabled on your behalf" notice

Fixes https://github.com/Automattic/dotcom-forge/issues/9238

## Proposed Changes

HEs sometimes enable Edge Cache Defensive Mode on behalf of users to protect their sites. This PR adds some UI to explain why defensive mode is suddenly enabled and hides the `Disable` button.

<img width="425" alt="Screenshot 2024-09-25 at 14 38 01" src="https://github.com/user-attachments/assets/859ba52d-88b4-4987-bdc4-47973d4bc401">

## Why are these changes being made?


See pdKhl6-4fY-p2

## Testing Instructions



1. Apply D162318-code and follow steps 1-3 in those testing instructions
2. Navigate to `/hosting-config/:site_slug`
3. Ensure that you see a notice in the `Defensive Mode` card containing the following text:

> Defensive mode was enabled on your behalf to protect your site.

4. Ensure that you don't see a `Disable` button in the `Defensive Mode` card
